zfsbootmenu/hook/zfsbootmenu-parse-commandline.sh

```diff
@@ -193,6 +193,13 @@ else
   zinfo "defaulting sort key order to ${zbm_sort}"
 fi
 
+# Allow hooks to be imported from another filesystem
+
+# shellcheck disable=SC2034
+if ! zbm_hook_root=$( get_zbm_arg zbm.hookroot ) ; then
```

Comment:
Can this be changed to more closely match the existing `get_zbm_arg` usage pattern?
```
zbm_hook_root=
if zbm_hook_root=$( get_zbm_arg zbm.hookroot) ; then
  zinfo "setting user hooks to ${zbm_hook_root}"
fi
```

Reply:
Done